server/routes/install-routes.js

```diff
@@ -6,9 +6,11 @@ const asyncHandler = require('../lib/express-async-handler');
 
 const { handleTracingMiddleware } = require('../tracing/tracing-middleware');
 const getVersionTags = require('../lib/get-version-tags');
+const preventClickjackingMiddleware = require('./prevent-clickjacking-middleware');
 
 function installRoutes(app) {
   app.use(handleTracingMiddleware);
+  app.use(preventClickjackingMiddleware);
```

Comment:
Tested and works to prevent embedding the archive in an iframe ✅:

```html
<iframe src="http://localhost:3050/"></iframe>
```

![](https://user-images.githubusercontent.com/558581/189248197-cdc4b8b9-cf64-4d38-9be8-8010eb4bfe2c.png)